MBIPS/MBIP-5.md

```diff
@@ -9,42 +9,41 @@ created: 2023-05-11
 
 ## Simple Summary
 
-Increases the used gas based on storage increase in transaction to deal wiuth storage congestion.
+Increases the gas used based on storage increase in transactions to deal with storage congestion.
 
 ## Abstract
 
-Introduce a new mechanism when executing a transaction that will increase the gas used when
-a transaction increases the storate stage.
+Introduce a new mechanism for executing transactions that increases the gas used when
```

Comment:
```suggestion
Introduce a new mechanism for executing transactions that increase the gas used when
```

Reply:
I think "increases" is right in this case, but not 100% sure